modules/hcp-configuring-oauth-console.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * hosted_control_planes/hcp-authentication-authorization.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="hcp-configuring-oauth-console_{context}"]
+= Configuring the OAuth server for a hosted cluster by using the web console
+
+You can configure the internal OAuth server for your hosted cluster by using the {product-title} web console. You can configure OAuth for the supported identity providers such as `oidc`, `htpasswd`, `keystone`, `ldap`, `basic-authentication`, `request-header`, `github`, `gitlab`, and `google`. Adding any identity provider in the OAuth configuration removes the default `kubeadmin` user provider.
+
+.Prerequisites
+
+* Log in as a user with `cluster-admin` privileges.
+* You created your hosted cluster.
+
+.Procedure
+
+. Navigate to *Home* -> *API Explorer*.
+
+. Use the *Filter by kind* box to search for your `HostedCluster` resource.
++
+image::config-oauth-console-hcp.png[Filtering the hosted cluster]
```

Comment:
[I don't think this screenshot is needed](https://redhat-documentation.github.io/supplementary-style-guide/#screenshots). I would recommend removing it because the part of the interface you are guiding them to is unambiguous and you have included very clear instructions about putting `HostedCluster` in the box that says "Filter by kind".

Reply:
Agreed